test/backend/BucketKeyTestSuite.scala

```diff
@@ -0,0 +1,80 @@
+package backend
+
+import com.scalableminds.util.geometry.Vec3Int
+import com.scalableminds.webknossos.datastore.models.{AdditionalCoordinateRequest, BucketPosition}
+import com.scalableminds.webknossos.datastore.models.datasource.AdditionalCoordinateDefinition
+import com.scalableminds.webknossos.tracingstore.tracings.volume.BucketKeys
+import org.scalatestplus.play.PlaySpec
+
+class BucketKeyTestSuite extends PlaySpec {
```

Comment:
```suggestion
class VolumeBucketKeyTestSuite extends PlaySpec {
```
(to hint that this is about volume annotations)